.github/actions/setup/action.yml

```diff
@@ -0,0 +1,68 @@
+name: Common setup
+
+inputs:
+  client-repo:
+    required: true
+    type: string
+  build-type:
+    required: true
+    type: string
+  target-device:
+    required: true
+    type: string
+  platform:
```

Comment:
Similar question as above. Is this host or target platform?

Reply:
Host platform.